### PR TITLE
Landing homepage component

### PR DESCRIPTION
**Pull request by saifudinbadani**
LandingPage and homepage added

Screen Shot 2022-04-06 at 2 16 27 pm

Screen Shot 2022-04-06 at 2 16 19 pm

### CODE REVIEW

File: src/components/Navigation/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+    return  <nav className="nav-container display-flex primary-clr p-rl-6">
+    <h1 to='/' className="nav-link font-size-2pt6 heading-1">Jotter</h1>
+    <div className="input-search pos-rltv m-rght-auto">
+        <i className="fas fa-filter pos-absolute font-size-2pt6"></i>
+        <input type="text" placeholder="search" className="nav-input-search"/>
+    </div>
+    {/* <Link to='/productlist' className="nav-link-cta">Shop Now</Link>
+    <Link to='/login' className="nav-link"><button
+            className="btn btn-solid-secondary">Login</button></Link> */}
```
Comment (vaishnavipaliya11) [inline src/components/Navigation/Navigation.js:14]: remove commented code

**Review by vaishnavipaliya11 — APPROVED** [review]

**Comment by vaishnavipaliya11** [comment]
Remove the commented code and you can proceed.
like the way you wrote description about specific component style its helps to understand.
you used semantic html which is good.

File: src/components/HomePage/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import './Homepage.css';
+import { Navigation } from "../Navigation/Navigation";
+import { SideBar } from './Sidebar';
+import { MainContent } from './MainContent';
+
+ export const HomePage = () => {
+
+    return <div className='homePage'>
+        <Navigation />
```
Comment (ivinayakg) [inline src/components/HomePage/HomePage.js:9]: add the navigation to the routes component that way you don't have to import the navigation component on every page

File: src/components/HomePage/Homepage.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+.main-notes-container{
+    flex-direction: column;
+    flex-grow: 1;
+    margin: 3.5rem 2rem;
+}
+/* Notes input */
+.input-note-container{
+    border: 1px solid black;
+    align-self: center;
+    border-radius: 0.3rem;
```
Comment (ivinayakg) [inline src/components/HomePage/Homepage.css:32]: use px if the cascaded value would be less than 10px

File: src/components/HomePage/MainContent.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            <input type='text' className='title-input-note heading-3 m-1' placeholder='title comes here...'/>
+            <textarea className='content-input-note m-1' placeholder='content comes here...'/>
+            <div className='notes-btn'>
+                    <i className="notes-icon fas fa-palette"></i>
+                    <i className="notes-icon fas fa-tag"></i>
+                    <button className='btn btn-solid-primary'>Save</button>
+                </div>
+        </div>
+        <div className='notes-container display-flex'>
+            <div className='note display-flex'>
```
Comment (ivinayakg) [inline src/components/HomePage/MainContent.js:16]: create this into a separate component

File: src/App.css [hunk 5]
```diff
@@ -3,3 +3,116 @@
...
+}
+
+.width-40{
+  width: 40rem;
+}
+
+/* utility class */
+
+.p-rl-6{
+  padding-left: 6rem;
```
Comment (parulgupta2001) [inline src/App.css:93]: you can use shorthand property here

File: src/components/HomePage/Sidebar.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import './Homepage.css';
+
+export const SideBar = () => {
+    return <aside className='side-wrapper display-flex'>
+        <button className='btn-sideWrapper  btn'>
```
Comment (ivinayakg) [inline src/components/HomePage/Sidebar.js:5]: for now its ok, but in the future use Link component from the react-router

**Review by ivinayakg — APPROVED** [review]
Good Work 👍

**Review by parulgupta2001 — APPROVED** [review]
consider the suggestion, otherwise everything looks good

File: src/components/HomePage/Sidebar.js (hunk 6 above)
Comment (saifudinbadani) [reply src/components/HomePage/Sidebar.js:5]: Thank you. I will do the suggested changes.

**Pull request merged**